explore/src/main/scala/explore/Routing.scala

```diff
@@ -27,16 +27,26 @@ object Routing {
         Observation.Id.fromString(s).map(ObsPage(_))
     }(p => p.obsId.format)
 
+  private val targetObsIdP: Prism[String, TargetPage] =
+    Prism[String, TargetPage] {
+      case s =>
+        Observation.Id.fromString(s).map(TargetPage(_))
+    }(p => p.obsId.format)
+
   val config: RouterWithPropsConfig[Page, View[RootModel]] =
     RouterWithPropsConfigDsl[Page, View[RootModel]].buildConfig { dsl =>
       import dsl._
 
       (emptyRule
         | staticRoute(root, HomePage) ~> renderP(view => HomeComponent(view))
-        | staticRoute("/constraints", ConstraintsPage) ~> render(UnderConstruction())
         | dynamicRouteCT(("/obs" / string("[a-zA-Z0-9-]+")).pmapL(obsIdP)) ~> renderP(view =>
           HomeComponent(view)
-        ))
+        )
+        | dynamicRouteCT(("/target" / string("[a-zA-Z0-9-]+")).pmapL(targetObsIdP)) ~> renderP(
```

Comment:
I don't recall if the order is important

Reply:
I don't think it is. I'm just keeping the same order everywhere (`AppTab`, `RootModelRouting` and here) for mental sanity.